jenkins-slaves/ansible/roles/common/tasks/main.yml

```diff
@@ -127,6 +135,102 @@
   with_items:
     - "{{ maven_versions }}"
 
+### GraalVM ###
+
+- name: Dowload GraalVM archives
+  get_url:
+    url: https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-19.3.1/graalvm-ce-java11-linux-amd64-19.3.1.tar.gz
```

Comment:
It would be great to extract the versions into constants to avoid repeating them in download, extract, etc. steps.